addons/web/static/src/js/views/view_dialogs.js

```diff
@@ -362,7 +362,7 @@ var SelectCreateDialog = ViewDialog.extend({
                 classes: "btn-default o_form_button_cancel",
                 close: true,
             }];
-            if (!self.options.no_create) {
```

Comment:
why adding a new option? I'd prefer to use no_create ->
`no_create: !self.can_create,` above